link_shortener/view_routes.py

```diff
@@ -104,6 +104,14 @@ async def delete_link_view(request, user, link_id):
                         status_code=str(status)
                     ), status=status)
 
+@view_blueprint.route('/delete/<link_id>/confirm/', methods=['GET'])
```

Comment:
/ navíc na konci endpointu. Myslim že to pak má v Sanicu jinou funkci. Nahoře v html to samé

Reply:
funguje to takisto s "/" aj bez, zmena pushnuta.